tests/core_tests.py

```diff
@@ -222,6 +261,13 @@ def test_api_v1_query_endpoint(self):
         resp = json.loads(self.get_resp("/api/v1/query/", {"query_context": data}))
         self.assertEqual(resp[0]["rowcount"], 100)
 
+    def test_api_v1_query_endpoint_with_post_processing(self):
+        self.login(username="admin")
+        qc_dict = self._get_query_context_dict_with_post_processing()
+        data = json.dumps(qc_dict)
+        resp = json.loads(self.get_resp("/api/v1/query/", {"query_context": data}))
```

Comment:
Do you want to change to `/api/v1/chart/data` ? if yes let's more the tests to the tests.charts.api_tests

Reply:
Will do once the new EP hits `master`.